src/compiler/crystal/codegen/abi.cr

```diff
@@ -33,7 +33,7 @@ abstract class Crystal::ABI
     end
   end
 
-  abstract def abi_info(atys : Array(LLVM::Type), rty : LLVM::Type, ret_def : Bool, context : Context)
+  abstract def abi_info(atys : Array(LLVM::Type), rty : LLVM::Type, ret_def : Bool, context : LLVM::Context)
```

Comment:
Now that we're validating these, it errored that it didn't know what `Context` is.